templates/plugin/ansible.yml.erb

```diff
@@ -6,3 +6,4 @@
 <% else -%>
 :working_dir: <%= scope.lookupvar("foreman_proxy::plugin::ansible::working_dir") %>
 <% end -%>
+:ansible_roles_path: <%= scope.lookupvar("foreman_proxy::plugin::ansible::roles_path") %>
```

Comment:
If you're always reading this fro the ansible.cfg and we ensure it's there, I would leave this out. It's only a risk of them getting out of sync.

Reply:
Don't both of them use the same variable? But I see your point about duplicity, removed.